src/components/NavBar.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
+import Game from './Game';
+import Home from './Home';
+
+function NavBar() {
+  return (
+    <Router>
+      <>
```

Comment:
since everything is contained inside Router, I guess you don't need a fragment here